test/org/rascalmpl/test/parser/ParserTest.java

```diff
@@ -152,9 +152,13 @@ public void testListSharing(){
 
 	public void testDoubleLeftNullable() {
 		executeParser(new DoubleLeftNullable());
-        }
+	}
+
+	public void testDoubleRightNullable() {
+		executeParser(new DoubleRightNullableWithPrefixSharing());
+	}
 
-        public void testDoubleRightNullable() {
```

Comment:
@arnoldlankamp why is this test not useful anymore?

Reply:
It's still there. This is just a tabs vs spaces formatting diff.